bcda/service/service.go

```diff
@@ -496,10 +496,15 @@ func (s *service) getBenesByFileID(ctx context.Context, cclfFileID uint, conditi
 			upperBound = time.Now()
 		}
 
-		ignoredMBIs, err = s.repository.GetSuppressedMBIs(ctx, s.sp.lookbackDays, upperBound)
-		if err != nil {
-			return nil, fmt.Errorf("failed to retreive suppressedMBIs %s", err.Error())
+		if cfg, ok := GetCtxACOCfg(ctx); ok {
+			if !cfg.IgnoreSuppressions {
```

Comment:
I assume this is covered by your tests as well as the bool type but just in case: if `cfg.IgnoreSuppressions` returns a string of either "true"/"false" then this block will always fail.

Reply:
IgnoreSuppressions will always be a bool type, so we should be OK here!